### PR TITLE
Remove <Render/> component

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
This PR removes the <Render/> component helper in favour of more classical react patterns to make the code base easier to understand for new contributors

Preview @ https://remove_render_comp.surge.sh

### CODE REVIEW

File: src/browser/modules/Stream/ParamsFrame.tsx [hunk 1]
```diff
@@ -47,14 +47,14 @@ const ParamsFrame = ({ frame }: any) => {
     typeof frame.success === 'undefined' ? null : (
       <StyledStatsBar>
         <Ellipsis>
-          <Render if={frame.success === true}>
```
Comment (jharris4) [inline src/browser/modules/Stream/ParamsFrame.tsx:50]: just a nit, but this could be combined into ternary, no?

**Comment by jharris4** [comment]
Looking good aside from a small nit. I don't see the commit to remove the Render component now that it's unused though?

**Review by jharris4 — APPROVED** [review]
LGTM

File: src/browser/modules/Stream/ParamsFrame.tsx (hunk 1 above)
Comment (OskarDamkjaer) [reply src/browser/modules/Stream/ParamsFrame.tsx:50]: I wasn't confident with this, with the weird "undefined" string check, so leaving it

**Pull request merged**